proto/mls/database/intents.proto

```diff
@@ -61,6 +61,22 @@ message RemoveMembersData {
   }
 }
 
+// The data required to make a commit that updates group membership
+// Handles both Add and Remove actions
+message UpdateGroupMembershipData {
+  // V1 of UpdateGroupMembershipPublishData
+  message V1 {
+    // Contains delta of membership updates that need to be applied
+    map<string, uint64> membership_updates = 1;
+    // Contains the list of members that will be removed
+    repeated string removed_members = 2;
```

Comment:
If the `membership_updates` contains delta, what are `removed_member` for?

Reply:
I thought about having an update that has a `sequence_id` of 0 be treated as a remove, but it felt a little hacky.